examples/gallery/line/vectors.py

```diff
@@ -0,0 +1,64 @@
+"""
+Cartesian, circular, and geographic vectors
+-------------------------------------------
+
+The :meth:`pygmt.Figure.plot` method can plot Cartesian, circular, and geographic vectors.
+The ``style`` parameter controls vector attributes.
+Vector documentation: https://www.pygmt.org/dev/gallery/line/vector-heads-tails.html
```

Comment:
Just a tiny edit here. We're still in the process of documenting the special `:doc:` linking functions at #802.
```suggestion
The ``style`` parameter controls vector attributes. See also
:doc:`Vector attributes documentation </gallery/line/vector-heads-tails>`.
```